R/cal-estimate.R

```diff
@@ -0,0 +1,405 @@
+#------------------------------ Estimates --------------------------------------
+#----------------------------- >> Logistic --------------------------------------
+#' Uses a logistic model to calibrate probabilities
```

Comment:
I would use the phrase "logistic regression" instead of "logistic model". There is more than one type of logistic model.